src/static/data/movie-source.json

```
@@ -0,0 +1,12 @@
+[
+  {
+    "sourceType": "NHK",
+    "name": "NHK For Schoolの動画",
+    "dataFile": "movies2-nhk.json"
```

```suggestion
    "dataFile": "movies-nhk.json"
```

あとあと、 `2` が意味不明になってしまいそうなので、 `2` は取りましょうか。jsonファイルもリネームしていただけると助かります。